packages/devtools_app/lib/src/screens/vm_developer/vm_function_display.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import 'vm_developer_common_widgets.dart';
+import 'vm_object_model.dart';
+import 'vm_service_private_extensions.dart';
+
+// TODO(mtaylee): Finish implementation of [ICDataArrayWidget] and add it to
+// the [VmFuncDisplay].
+
+/// A widget for the object inspector historyViewport displaying information
+/// related to 'Func' (function) objects in the Dart VM.
+class VmFuncDisplay extends StatelessWidget {
+  const VmFuncDisplay({
+    required this.function,
+  });
+
+  final FuncObject function;
+
+  @override
+  Widget build(BuildContext context) => VmObjectDisplayBasicLayout(
```

Comment:
Nit: @kenzieschmoll probably has an opinion on this, but I seem to recall that we prefer to only use `=>` notation if the right hand side fits on a single line.

Reply:
+1. Adding brackets, especially for Flutter build methods, results in easier to read code and more consistent formatting.